### PR TITLE
Feature/fusion components/button relative url

### PR DESCRIPTION
**Pull request by thomasFjorstad**

### CODE REVIEW

File: src/components/general/Button/components/Anchor.tsx [hunk 1]
```diff
@@ -16,30 +18,49 @@ type AnchorProps = {
...
+    ) => {
+        if (Boolean(relativeUrl)) {
+            return (
+                <span ref={ref}>
+                    <Link
+                        to={"/123"}
+                        className={getButtonClasses(props)}
+                        target={targetBlank ? '_blank' : '_self'}
+                        onMouseDown={onMouseDown}
+                        onMouseUp={onMouseUp}
```
Comment (thomasFjorstad) [inline src/components/general/Button/components/Anchor.tsx]: Not sure why they have used onClicked and onClickedCapture in <a />

File: src/components/general/Button/components/Anchor.tsx [hunk 2]
```diff
@@ -16,30 +18,50 @@ type AnchorProps = {
...
+        }
+        else {
+            return (
+                <a
+                    className={getButtonClasses(props)}
+                    href={url}
+                    target={targetBlank ? '_blank' : '_self'}
+                    onMouseDown={onMouseDown}
+                    onMouseUp={onMouseUp}
+                    onClick={onClick}
```
Comment (thomasFjorstad) [inline src/components/general/Button/components/Anchor.tsx]: Not sure why they have used onClicked and onClickedCapture in <a />

File: src/components/general/Button/__stories__/Button.stories.tsx [hunk 3]
```diff
@@ -14,6 +14,18 @@ const createButtonStory = props => () => (
         <Button disabled {...props} {...eventsFromNames}>
             Disabled button
         </Button>
+
+        <div>
```
Comment (sandves) [inline src/components/general/Button/__stories__/Button.stories.tsx]: I do not see what value these are providing to the story book. The button texts are also lying as neither the `url` prop or the `relativeUrl` prop is passed to the component. Maybe just remove them?

File: src/components/general/Button/components/Anchor.tsx [hunk 4]
```diff
@@ -16,30 +18,50 @@ type AnchorProps = {
...
-            onMouseUp={onMouseUp}
-            onClick={onClick}
-            onClickCapture={onClickCapture}
-            ref={ref}
-        >
-            <Content>{children}</Content>
-        </a>
-    )
+    ) => {
+        if (Boolean(relativeUrl)) {
```
Comment (sandves) [inline src/components/general/Button/components/Anchor.tsx]: Should not be necessary to parse this to boolean explicitly I think.

File: src/components/general/Button/components/Anchor.tsx [hunk 5]
```diff
@@ -16,30 +18,50 @@ type AnchorProps = {
...
+            return (
+                <span
+                    className={getButtonClasses(props)}
+                    ref={ref}>
+                    <Link
+                        to={"/123"}
```
Comment (sandves) [inline src/components/general/Button/components/Anchor.tsx]: Should probably use the `relativeUrl` here?

File: src/components/general/Button/__stories__/Button.stories.tsx (hunk 3 above)
Comment (thomasFjorstad) [reply src/components/general/Button/__stories__/Button.stories.tsx]: Done

File: src/components/general/Button/components/Anchor.tsx (hunk 5 above)
Comment (thomasFjorstad) [reply src/components/general/Button/components/Anchor.tsx]: Ups, a bit fast there ;p

File: src/components/general/Button/components/Anchor.tsx (hunk 4 above)
Comment (thomasFjorstad) [reply src/components/general/Button/components/Anchor.tsx]: Removed `Boolean`

**Pull request merged**